src/main/java/com/autotune/experimentManager/ExperimentManager.java

```diff
@@ -68,5 +67,6 @@ public static void notifyScheduledQueueProcessor() {
     private static void addEMServlets(ServletContextHandler context) {
         context.addServlet(CreateExperimentTrial.class, EMConstants.APIPaths.CREATE_EXPERIMENT_TRIAL);
         context.addServlet(ListTrialStatus.class, EMConstants.APIPaths.LIST_TRIAL_STATUS);
+        context.addServlet(InitiateExperimentTrial.class, EMConstants.APIPaths.EXPERIMENT_MANAGER_INITIATE_EXPERIMENT_TRAIL);
```

Comment:
Should TRIAL instead of TRAIL -> `EMConstants.APIPaths.EXPERIMENT_MANAGER_INITIATE_EXPERIMENT_TRIAL`